src/test/java/rv/atqc/gttesting/editAvatarURL/EditAvatarURLMozilla.java

```diff
@@ -26,7 +27,7 @@
     @BeforeMethod
     public void before() {
         FirefoxDriverManager.getInstance().setup();
-        driver = new FirefoxDriver();
+        driver = DriverFactory.init(DriverFactory.FIREFOX);
```

Comment:
Why you are using DriverFactory.init(DriverFactory.FIREFOX); if you have Firefox class?